### ISSUE TITLE
Persist action messages and provide a view

### ISSUE THREAD

**Issue by csarven** [dokieli/dokieli#385]
Some kinds of activities trigger a notification that alerts (e.g., via `showActionMessage`) the user on what's happening, e.g., document mode changing, encountering information that's of interest to the user, opening a resource, content being copied to clipboard, creating an archive snapshot, notification being received, URL that can be copied.

These messages appear for a few seconds and than disappear. For accessibility and reuse, it should be possible for the user to see a log of these notifications with timestamps and the activity as before. This could be accessible from the dokieli menu.

**Comment by csarven** [comment]
>This could be accessible from the dokieli menu.

is implemented and currently accessible via "Message Log"
